styles/dialog/fight.scss

```diff
@@ -37,17 +95,38 @@
                 font-weight: bold;
                 padding-left: 5px;
             }
-            .control-button {
-                width: 49%;
-                height: min-content;
+            .portrait-actions {
+                flex-grow: 1;
+                display: flex;
+                //justify-content: center;
+                //align-items: center;
+                .fight-rolls {
+                    flex-grow: 1;
+                    display: flex;
```

Comment:
As I'm specifying more flex options here I think it would be clearer to have them all in one place so it's much easier to read and leverage. flex-column is just an alias for some of those properties.